bsp/nuclei/gd32vf103_rvstar/board/board.c

```diff
@@ -32,6 +32,125 @@ extern void *_heap_end;
  */
 extern void _init(void);
 
+// Check pin assignment here https://doc.nucleisys.com/nuclei_board_labs/hw/hw.html
+
+void rt_hw_spi_drvinit(void)
+{
+    // rcu_periph_clock_enable(RCU_GPIOA);
```

Comment:
What's that? If these code is disable, please remove them.

Reply:
I leave these commented lines to be references for user who want to add pinmux configurations.

Reply:
I have pushed some changes to optimize the code comment of board.c